### PR TITLE
Service: Sound player

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/soundplayer.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Sound player
+
+A device that can play various sounds stored locally. This service is typically paired with a ``storage`` service for storing sounds.
+
+    identifier: 0x1e3048f8
+
+## Registers
+
+    rw volume: u16 @ intensity
```
Comment (mmoskal) [inline services/soundplayer.md:9]:
```suggestion
    rw volume: u16 / @ intensity
```

File: services/soundplayer.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Global volume of the output. ``0`` means completely off. This volume is mixed with each play volumes.
+
+    const supported_wav_format? @ 0x80 {
+        stereo: bool
+        bit: u8
```
Comment (mmoskal) [inline services/soundplayer.md:15]:
```suggestion
        bits_per_sample: u8
```

File: services/soundplayer.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    identifier: 0x1e3048f8
+
+## Registers
+
+    rw volume: u16 @ intensity
+
+Global volume of the output. ``0`` means completely off. This volume is mixed with each play volumes.
+
+    const supported_wav_format? @ 0x80 {
+        stereo: bool
```
Comment (mmoskal) [inline services/soundplayer.md:14]:
```suggestion
        num_channels: u8
```

File: services/soundplayer.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        bit: u8
+        max_sample_rate: u16 Hz
```
Comment (mmoskal) [inline services/soundplayer.md:16]: I think this should u32, first field for alignment. There may be situations where you want more than 65kHz

File: services/soundplayer.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        name: string
+    }
+
+Starts playing a sounds with a specific volume.
+
+    command sounds @ 0x81 {
+        sounds_port: pipe
+    }
+    pipe report sounds_pipe {
+       duration: u16 s
```
Comment (mmoskal) [inline services/soundplayer.md:34]:
```suggestion
       duration: u32 ms
```

File: services/soundplayer.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+    identifier: 0x1e3048f8
+
+## Registers
+
+    rw volume: u16 / @ intensity
+
+Global volume of the output. ``0`` means completely off. This volume is mixed with each play volumes.
+
+    const supported_wav_format? @ 0x80 {
```
Comment (pelikhan) [inline services/soundplayer.md:13]: this should go somewhere

**Pull request merged**